### ISSUE TITLE
Replit breaks with upgraded dependencies, nix channel, and node

### ISSUE THREAD

**Issue by sz6084** [titaniumnetwork-dev/Ultraviolet-App#21]
## Expected Behavior

Once imported from Github to Replit, will run, open main page(index.html), and unblock websites

## Current Behavior

Does not run and gives error in console:
![image](https://user-images.githubusercontent.com/101011086/214465844-dc4b852b-dd46-46b8-aeb4-c2704e03c82e.png)

## Possible Solution

Revert upgraded dependencies, nix channel, and node. Try looking to see if Replit currently supports these versions

## Steps to Reproduce

Example: (https://replit.com/@sz6084/Ultraviolet-App)

1. If not already, have an imported Ultraviolet App
2. Pull any new changes/updates using the Git page.
3. Try running the Repl

## Context (Environment)

I'm trying to run the Repl to unblock a website. Pulling recent changes from Git page causes incompatability issues that may be hard to revert.

## Detailed Description

Check if Replit supports the version of the upgraded dependencies, nix channel, and node

*If there is another issue with my imported project please let me know.

**Comment by e9x** [comment]
Perhaps I entered the NIX channel incorrectly. I will look into this, thanks.
